hax/hax/motr/__init__.py

```diff
@@ -110,6 +112,36 @@ def _entrypoint_request_cb(self, reply_context: Any, req_id: Any,
                   " '{}', process fid = {}".format(remote_rpc_endpoint,
                                                    str(process_fid)) +
                   ' The request will be processed in another thread.')
+        try:
+            cns = ConsulUtil()
```

Comment:
`ConsulUtil` will be created every time a new entrypoint request comes which is probably not the best idea.

Please also note that `self.consul_util` is accessible here. BTW I guess it was you who added it 😉

Reply:
I think there's some problem in merging to main here. I overwrote the earlier version. Let me fix this.

Reply:
Fixed.